src/widgets/partnered/ui/partnered.module.scss

```diff
@@ -0,0 +1,41 @@
+.partnered-content {
+  display: flex;
+  flex-direction: column;
+
+  .title {
+    font-size: 36px;
+    font-weight: $font-weight-medium;
+    line-height: 44px;
+    color: $color-black;
+    letter-spacing: 0;
+  }
+
+  .partners {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    align-self: center;
+    justify-content: space-between;
+
+    width: 100%;
+    max-width: 500px;
+    padding: 16px;
```

Comment:
We can give up padding here. To save the layout we can add a gap between the header and the list.